### DISCUSSION TITLE
Multiple columns for setDefaultSort

### DISCUSSION THREAD

**Discussion by JimKarvo** [rappasoft/laravel-livewire-tables#1664] [Q&A]
Hello,
I am trying to have a default behavior for the `$this->setDefaultSort();` in order to include the `created_at` and the `name` for query

I want, if the are more than oe rows at created_at with same date, to also sort that values with name, instead of (default) id.

Thank you

**Comment by lrljoe** [comment]
So, multiple columns in default sort did present some issues when we previously introduced it (in v2).

Please can you raise an "Issue" for this, marked as a Feature Request, so that we can track it behind the scenes and not lose track of it.
